pkg/controller/logstorage/linseed/linseed_controller.go

```diff
@@ -374,19 +409,22 @@ func (r *LinseedSubController) Reconcile(ctx context.Context, request reconcile.
 	}
 
 	cfg := &linseed.Config{
-		Installation:      install,
-		PullSecrets:       pullSecrets,
-		Namespace:         helper.InstallNamespace(),
-		BindNamespaces:    bindNamespaces,
-		TrustedBundle:     trustedBundle,
-		ClusterDomain:     r.clusterDomain,
-		KeyPair:           linseedKeyPair,
-		TokenKeyPair:      tokenKeyPair,
-		UsePSP:            r.usePSP,
-		ESClusterConfig:   esClusterConfig,
-		HasDPIResource:    hasDPIResource,
-		ManagementCluster: managementCluster != nil,
-		Tenant:            tenant,
+		Installation:        install,
+		PullSecrets:         pullSecrets,
+		Namespace:           helper.InstallNamespace(),
+		BindNamespaces:      bindNamespaces,
+		TrustedBundle:       trustedBundle,
+		ClusterDomain:       r.clusterDomain,
+		KeyPair:             linseedKeyPair,
+		TokenKeyPair:        tokenKeyPair,
+		UsePSP:              r.usePSP,
+		ESClusterConfig:     esClusterConfig,
+		HasDPIResource:      hasDPIResource,
+		ManagementCluster:   managementCluster != nil,
+		Tenant:              tenant,
+		ElasticHost:         elasticHost,
+		ElasticPort:         elasticPort,
+		ElasticClientSecret: esClientSecret,
```

Comment:
If we are using mTLS, this secret also contains the CA, right ? `ElasticClientSecret`makes me think we only store the client certificate.

Reply:
The CA is not within this secret. The CA is contained within the trusted bundle that's already provided, so you're correct that this only contains the client certificate and key. 

The comment on the field explains this also: https://github.com/tigera/operator/blob/0ec02e4228f794c340c5f9f7440129575d9aa70c/pkg/render/logstorage/linseed/linseed.go#L120-L122